pkg/scannerv4/mappers/mappers.go

```diff
@@ -666,21 +666,30 @@ func cvssMetrics(_ context.Context, vuln *claircore.Vulnerability, nvdVuln *nvds
 		preferredCVSS, preferredErr = vulnCVSS(vuln, v4.VulnerabilityReport_Vulnerability_CVSS_SOURCE_OSV)
 	case strings.EqualFold(vuln.Updater, constants.ManualUpdaterName):
 		// It is expected manually added vulnerabilities only have a single link.
-		preferredCVSS, preferredErr = vulnCVSS(vuln, sourceFromLinks(vuln.Links))
+		src := sourceFromLinks(vuln.Links)
+		// When both NVD and manual data have available CVSS scores, use the NVD score.
+		// And only the NVD score is included in the metrics, as the manual data originates from the same
+		if src == v4.VulnerabilityReport_Vulnerability_CVSS_SOURCE_NVD {
+			cvss, cvssErr := nvdCVSS(nvdVuln)
```

Comment:
can you also update the `func cvssMetrics(_ context.Context, vuln *claircore.Vulnerability, nvdVuln *nvdschema.CVEAPIJSON20CVEItem) ([]*v4.VulnerabilityReport_Vulnerability_CVSS, error)` above to `func cvssMetrics(_ context.Context, vuln *claircore.Vulnerability, nvdVuln nvdschema.CVEAPIJSON20CVEItem) ([]*v4.VulnerabilityReport_Vulnerability_CVSS, error)` (ie change `nvdVuln` to not be a pointer). It's mildly unrelated, but `nvdCVSS` assumes it's not `nil` and in theory it could be (in practice it's not). I'd like to remove the theory and just ensure it's never `nil`. Thanks